metering/metering-upgrading-metering.adoc

```diff
@@ -0,0 +1,32 @@
+[id="upgrading-metering"]
+= Upgrading Metering
```

Comment:
s/Metering/metering/ ?
To be consistent with other topics and the lowercase usage in this assembly.